switch to using getOwnPropertyNames, but ignore standard function sta…

bringking opened the pull request:
…tic methods

What about something like this to fix the issue for now?  Do you see any downsides to this implementation?

bringking commented:
Relates to - https://github.com/yahoo/fluxible/issues/235

mridgway commented on index.js:
```diff
@@ -15,10 +15,20 @@ var REACT_STATICS = {
+    name: true,
+    length: true,
+    prototype: true,
+    displayName: true,
```
This looks like a duplicate from REACT_STATICS.

mridgway commented on .gitignore:
```diff
@@ -2,3 +2,6 @@
 npm-debug.log
+
+# Intellij
+.idea/
```
This shouldn't be in the project .gitignore, but your personal .gitignore. Otherwise this file would be full of ignores for every editor that contributors use. Please remove the changes in this file.

mridgway commented:
This makes sense :+1:. Couple of minor comments to address and then we'll merge.

bringking commented:
Cool, I removed the gitignore stuff and removed the duplicate displayName property from the KNOWN_STATICS object. Is that what you were referencing, or did you want the keys merged into one object?

mridgway commented:
You nailed it. Thanks!
